aws-test/tests/aws_glue_catalog_database/test-get-query.sql

```diff
@@ -0,0 +1,3 @@
+select name, title, akas
```

Comment:
can we query other important columns like `catalog_id` , `create_table_default_permissions`, `location_uri`, `description` in get call?